src/amo/components/AddonMeta/index.js

```diff
@@ -23,7 +24,7 @@ export class AddonMetaBase extends React.Component<Props> {
     const addonRatingCount = addon && addon.ratings ?
       addon.ratings.count : null;
     const averageDailyUsers = addon ? addon.average_daily_users : null;
-
+    const reviewUrl = addon && addon.slug ? `/addon/${addon.slug}/reviews` : '/';
```

Comment:
This shouldn’t link to anything if there is no addon as that’s the loading state. I think the reviews URL should have a trailing slash as well.